### ISSUE TITLE
Create Help page re caveats on search results

### ISSUE THREAD

**Issue by PatReynolds** [FreeUKGen/FreeBMD2#129]
See Barrie's comments on https://docs.google.com/document/d/18ZvMPbsmeYsrRcjnOI-NQ6MKdH3GpmFHEQH3ExZNuxQ/edit?usp=sharing

**Comment by richpomfret** [comment]
@PatReynolds this would be a good story to nudge forward over the next sprint if we can.

**Comment by PatReynolds** [comment]
Text is now on https://test2.freebmd.org.uk/cms/help.  Will need changing in order to add impage of the 'submit correction' button in location, and to use correct icon for "view original" once these are finalised/created.

**Comment by PatReynolds** [comment]
Moving to In Progress and assigning to Documentation Team

**Comment by PatReynolds** [comment]
Current system is  reached from the search detail page (ie as sytem planned here), and provides detailed instructions (which seem not to be read by some users) - https://www.freebmd.org.uk/cgi/corrections.pl?r=252876913&d=bmd_1593465254 and a form that contains the details from the entry in the detailed results page and provides the user with an opportunity to correct those details in a form.

A suggestion:
The 'report a data problem' takes the user to a number of option - each option is based on the user identifying what it is they want to do - and then receiving the information (or the form) rather than having a long text to read:
e.g "I want to change something to match what is on the certificate (or family bible or FreeREG, FreeCEN ..." does not go to a form but to an explanation.
I have divided the current explanations and form on this spreadsheet 
https://docs.google.com/spreadsheets/d/1VQt3Fjl1y2eixHsPQVp2gSZ0VEVPZ4xYsisbkLiZ01c/edit?usp=sharing

Please discuss on Slack whether this structure/pathway is what is needed, and on the spreadheet if it is (I have largely used the FreeBMD1 text, with some changes for clarity when seen in isolation and/or adding in 'next step'.

**Comment by PatReynolds** [comment]
Basic model seems to meet approval, so I suggest it is implemented, and the text reviewed/tweaked, particularly by the FreeBMD1 volunteers who respond to support queries and those who process amendments submitted by the form, and as a result of beta testing.

**Comment by PatReynolds** [comment]
Technical note: the error reporting page contains a form which pre-populates with the current record and allows the user to comment on each one, as appropriate.   Is this possible for MVP, or can we only achieve the URL (the "citation" in FreeBMD1 and a list of fields for the person to comment in?

**Comment by PatReynolds** [comment]
Adding @ComfE-2662 (please ask me if not clear!)

**Comment by richpomfret** [comment]
Any update @ComfE-2662?

**Comment by ComfE-2662** [comment]
Hi Rich,

Sorry I haven't got round to checking this as I have been booked out for work since furlough finished.

Will be able to update before the next meeting.

Comfort

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by PatReynolds** [comment]
@PhilipClarke712 is this something you could take a look at? Thanks, Pat

**Comment by PhilipClarke712** [comment]
What am I being asked to do: (1) combine the page that Barrie refers to at the very top with the current contractor-written Help page on test3 to make a new Help page (I don't have editing rights for test2, so I'll use what is on test3); **or** (2) to comment on doing such a thing?

Note: I don't particularly like that new Help page so I have another two copies of it on test3. The first one will just be adding hyperlinks and the second one will be adding back links to the old FreeBMD help sub-pages that were moved across to test3 (and being rewritten in active tense) which the current Help "ignores" , having copied what it wanted out of them

**Comment by PatReynolds** [comment]
Hi[ Philip, thanks for this - my comment on the other Help page story may be superfluous.  I've changed the titel of this story so it no longer refers to the help page.  This is about reporting errors (and speeding up the solution by providing information rather than letting the person send a message asking for help - although they can of course still do that as a last resort.

https://docs.google.com/spreadsheets/d/1VQt3Fjl1y2eixHsPQVp2gSZ0VEVPZ4xYsisbkLiZ01c/edit?usp=sharing

Is the text for the  Data Error Report Page OK?  This will be hard coded by Vino (the page is reachable from the text 'I have viewed the scan and wish to submit a correction' on an Entry Information page.

**Comment by PhilipClarke712** [comment]
@PatReynolds ,
I don't have time today, but I will look at it tomorrow (my energy supplier has decided to change both my electricity and gas meters on Friday a.m.: so I'm trying to accommodate no electricity, which means no internet, and no gas for a couple of hours - if they don't mess up).

**Comment by PatReynolds** [comment]
Many thanks, Philip! WE had our electricity out for a few hours last week
and it was a bit of a trial - but we had matches to light the gas stove, so
were well ahead of you.  Hope all goes well,

Pat

**Comment by PhilipClarke712** [comment]
Hi @PatReynolds,
I added various comments, all in the form of agreement with FreeBMD's commentators or adding further information / comment. I have no disagreement with the polices stated by FreeBMD. 

Note: the two meters were changed without too much disruption (in just over two hours total): but only the electricity is Smart, either the gas meter refuses to talk to electric meter (which is the "master") or it was never activated - but I have gas.

**Comment by PatReynolds** [comment]
Thanks, @PhilipClarke712 - please let me know if you want any advice/help/checking on this one.

**Comment by PhilipClarke712** [comment]
Hi @PatReynolds,
I did look at the spreadsheet on 1st February and I left six or seven comments in column C, but it appears that I totally forgot to state here that I had done so. Sorry about that.

**Comment by PhilipClarke712** [comment]
Hi @PatReynolds 

Having looked now at my comments: I have not raised any objections to what is there already, but have added (only in the comments)  explanations or suggestions to add a bit more text.  What would you like me to do now: I could copywrite the text, if you like? Note: I am lead to believe that it will be a programming implementation not an html page?

**Comment by PhilipClarke712** [comment]
Hi @PatReynolds 
I only have comment-rights to https://docs.google.com/spreadsheets/d/1VQt3Fjl1y2eixHsPQVp2gSZ0VEVPZ4xYsisbkLiZ01c/edit#gid=0 so I have expanded on my comments of 1st February and in some cases add in the comment suggested extra text.
These are to be implemented via buttons, and I don't know what constraints there are, if any, on how much text a button can accommodate.  Does, this meant expectations?

**Comment by PatReynolds** [comment]
Hi @PhilipClarke712 with comment rights you can edit the text, and it will show the original  (crossed out) as well as your revision.
Is that sufficient for your needs?

**Comment by PhilipClarke712** [comment]
Hi @PatReynolds ,
Thanks, Yes that will be fine.

**Comment by richpomfret** [comment]
@DeniseColbert to check if this story is still needed or not.

**Comment by richpomfret** [comment]
Agreed plan to review the linked document - decide what/any stories may arise from the comments.

**Comment by PatReynolds** [comment]
New created - moving to on ice until ready for review
